docker/candlepin-base/cp-test

```diff
@@ -288,10 +286,27 @@ else
     fi
 fi
 
-# Check if we need to use Java 11, otherwie auto detect Java version.
-# Candlepin support Java 11 from 3.2.0 & onwards.
 
-CANDLEPIN_BASE_VERSION=$(grep "Version:" server/candlepin.spec.tmpl | awk -F"Version: " '{print $2}')
+# Candlpin project restructuring
+# We need to correctly set the project home path.
+# We define CP_NEW_STRUCTURE boolean to check if
+# we are running with new candlepin project structure.
+CP_NEW_STRUCTURE=true
+PROJECT_DIR="."
+echo "Current branch : $CHECKOUT"
+echo "Target branch : $TARGET"
+
+if [ "$TARGET" != "master" ]; then
```

Comment:
I just run a 4.1.1 release, so we should probably use 4.1.2 here instead

Reply:
Since we fetch the version from ```candlepin.spec.tmpl``` file. It will be better if we check that this file exists on the root or not.
Something like - 
```
if [ ! -f candlepin.spec.tmpl ]; 
   Old Structure
else
    New Structure
```